explore_the_world/ui/utils.py

```diff
@@ -32,7 +32,6 @@ def haystack_is_ready():
     return False
 
 
-@st.cache
```

Comment:
Don't have much of an opinion here so just want to ask: are you sure? :)

Reply:
I also don't have an opinion, I just followed the instructions of the error message I got :smile: Might be worth double checking: add it back, try to upload a broken file and see if you also get the ugly deprecation error in the main window. If you don't then we can keep this line